htdocs/fichinter/card.php

```diff
@@ -882,6 +882,7 @@
 
 			$soc = $objectsrc->thirdparty;
 
+			$ref_client = (!empty($objectsrc->ref_client) ? $objectsrc->ref_client : GETPOST('ref_client', 'alpha'));
```

Comment:
Imagine you crane an intervention from a proposal.
The proposal as the ref PRABC for you and the proposal as the ref PR456 on customer side (so ref_client is PR456.

If you copy the ref of proposal into the intervention, you will got the ref of intervention that is
IN123 and the reference of this intervention into the customer system of the client will ba saved as PR456.
But this is a reference of a proposal not of an intervention. You are mixing apple and pear here. So i suggest you introduce a hidden option
INTERVENTION_SET_CUSTOMER_PROPOSAL_REF_WITH_CUSTOMER_INTERVENTION_REF